src/libraries/libraries-packages.proj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.Build.Traversal" DefaultTargets="Pack">
 
   <PropertyGroup>
-    <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
+    <TraversalGlobalProperties>BuildAllProjects=true;BuildTargetFramework=</TraversalGlobalProperties>
```

Comment:
This one is required to enable building of packages after just a vertical build. i.e.

build -s clr.corelib+libs
dotnet build libraries-packages.proj /p:BuildAllConfigurations=true

becuase if buildtargetFramework is set then we wont build the all the configurations of the package.

Reply:
Right, i might have missed something.